drivers/shared/executor/executor_test.go

```diff
@@ -289,7 +318,7 @@ func TestUniversalExecutor_LookupPath(t *testing.T) {
 // than mounting the underlying host filesystem
 func setupRootfs(t *testing.T, rootfs string) {
 	paths := []string{
-		"/bin/sh",
+		//		"/bin/sh",
```

Comment:
Why did you remove /bin/sh? Shouldn't it be permanently removed?

Reply:
Oh drat this change wasn't suppose to make it in.